libraries/rush-lib/src/api/RushProjectConfiguration.ts

```diff
@@ -349,10 +349,9 @@ export class RushProjectConfiguration {
   ): string | undefined {
     const rushConfiguration: RushConfiguration | undefined = this.project.rushConfiguration;
     if (rushConfiguration) {
-      const rushConnect: RushConnect = RushConnect.loadFromLinkStateFile(rushConfiguration);
-      const subspaceNameList: string[] = Object.keys(rushConnect.rushLinkState ?? {});
-      if (subspaceNameList.includes(this.project.subspace.subspaceName)) {
-        return 'Caching has been disabled for this project because it is a linked package.';
+      const rushConnect: RushConnect = RushConnect.loadFromRushConfiguration(rushConfiguration);
+      if (rushConnect.hasAnyLinksInSubspace(this.project.subspace.subspaceName)) {
+        return 'Caching has been disabled for this project because it is in a subspace that has a link.';
```

Comment:
_"Caching has been disabled for this project because it is in a subspace that has a link."_

Has a link?  What kind of link?

Maybe this feature would benefit from a special terminology like "hotlink" or "live link" etc.  The key concepts being:

1. The symlink is temporary, for testing purposes, and reset when you run "rush install" again.
2. The symlink is not reflected in the lockfile or committed to Git.
3. The symlink may violate SemVer expectations.

Reply:
Sample usage of these terms:

_"The `rush link-package` and `rush bridge-package` commands both facilitate testing locally built changes by creating hotlinks to library project folders."_

_"Caching has been disabled for this project because it is in a subspace that has a hotlink.""_